README.md

```diff
@@ -37,7 +76,7 @@ The file viewer.html contains several endpoints to allow data to be be exported
 
 `getDataURL` - **\[optional]** the location of where you have previously stored the user's partially completed interview answer data. This is optional and only needed if you are setting up your website to remember the data that a user has entered so far. For example this is useful in cases such as where you let people register at your website and collect information that you can use in all interviews on your website and information that might be common to all interviews (like name, address, etc.). At present, this DOES NOT mean that the user can save the interview at a particular point and then come back to that same point. When they return to the interview later, they must start from the beginning - with the only advantage being that the data they had previously entered is already filled in.
 
-`setDataURL` - **\[optional]** location of where you want to send the data from the interview when the use SENDs or SUBMITs the completed interview. This must be some website-based program that can "catch" the XML data stream that contains the user's answers to your interview in HotDocs .anx format.
+`setDataURL` - **\[optional]** location of where you want to send the data from the interview when the use SENDs or SUBMITs the completed interview. This must be some website-based program that can "catch" the XML data stream that contains the user's answers to your interview in HotDocs .anx format and in an equivalent json format through an http POST.
```

Comment:
user or end user

Reply:
I don't understand this comment.

Reply:
highlighted the wrong stuff `.... when the use SENDs or SUBMITs`  seems like `use` should be `user`

Reply:
fixed in later commit